code/Common/src/main/java/net/selenate/common/comms/req/SeReqElementGetAttribute.java

```diff
@@ -0,0 +1,54 @@
+package net.selenate.common.comms.req;
+
+import net.selenate.common.SelenateUtils;
+import net.selenate.common.comms.SeElementSelector;
+
+public class SeReqElementGetAttribute implements SeCommsReq {
+  private static final long serialVersionUID = 45749879L;
+
+  private final SeElementSelector selector;
+  private final String            attributeName;
```

Comment:
Is there a use case where you need to fetch multiple attributes?
If so, it would be better to provide a list of attribute names here.